frontend/src/components/explorer/mapViewer/DataOverlayValidation.vue

```diff
@@ -15,13 +15,15 @@
 </template>
 
 <script>
+import { default as messages } from '@/content/messages';
+
 export default {
   name: 'DataOverlayValidation',
   methods: {
     validateFile(e) {
       if (e.target.files.length !== 0) {
         if (e.target.files[0].type !== 'text/tab-separated-values') {
```

Comment:
It seems fileType checking is not working now. When I try to upload a non-TSV file, either a textfile or binary file, the error message is not shown.

Reply:
Ah good catch! I'll look at it :+1:

Reply:
The bug is fixed in commit 35878b1f76a52469954cf891d64acb535c17d45a